backend/.rubocop.yml

```
@@ -1,54 +1,1169 @@
+  Enabled: true
+
+Performance/UnneededSort:
+  Enabled: true
+
+Performance/UriDefaultParser:
+  Enabled: true
+
+Performance/ChainArrayAllocation:
+  Enabled: true
+
+Performance/CollectionLiteralInLoop:
```

YAMLlint (1.37.1)

[error] duplication of key "Performance/CollectionLiteralInLoop" in mapping

(key-duplicates)